apis/Google.Cloud.PubSub.V1/Google.Cloud.PubSub.V1/SubscriberClient.cs

```diff
@@ -81,9 +81,25 @@ public enum Reply
     /// </summary>
     public static TimeSpan MinimumAckExtensionWindow { get; } = TimeSpan.FromMilliseconds(50);
 
+    /// <summary>
+    /// The default minimum message Acknowledgement deadline extension of 0 seconds.
+    /// </summary>
+    public static TimeSpan DefaultMinimumAckDeadlineExtension { get; } = TimeSpan.FromSeconds(0);
+
+    /// <summary>
+    /// The default maximum message Acknowledgement deadline extension of 0 seconds.
```

Comment:
That suggests we can never extend at all - should this be an hour or something like that?